sdk/typescript/src/types/transactions.ts

```diff
@@ -395,6 +395,8 @@ export type SuiTransactionBlockResponse = Infer<
 export const SuiTransactionBlockResponseOptions = object({
   /* Whether to show transaction input data. Default to be false. */
   showInput: optional(boolean()),
+  /* Whether to show transaction raw input data. Default to be false. */
+  showRawInput: optional(boolean()),
```

Comment:
Do you also need to add the type to the response structure?

Reply:
Yes, without adding the field to the response structure, any request that uses this newly added option will fail the type validation

Reply:
Got it, I wanted to add this b/c I saw error like
```
"SuiTransactionBlockResponseQuery": InvalidParams(invalid array length 20, expecting 32 at line 1 column 70
```
[loki](https://metrics.sui.io/explore?orgId=1&left=%7B%22datasource%22:%22CU1v-k2Vk%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bhost%3D%5C%22ewr-tnt-rpc-11%5C%22%7D%20%7C%3D%20%60%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22CU1v-k2Vk%22%7D,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D) heard from Chris that this is not the root cause, I will close this PR then.